charts/rota-slackbot/templates/ingress.yaml

```
@@ -58,4 +58,7 @@ spec:
               {{- end }}
           {{- end }}
     {{- end }}
```

I think path should go inside host loop

```suggestion
          {{- if .Values.ingress.extraPaths }}
          {{- toYaml .Values.ingress.extraPaths | indent 10 }}
          {{- end }}
    {{- end }}
```

---

In that case, every host will get these extra paths added to it. Is this intended to be so?
Also, line 61 refuses to work like this (except if the `-` is removed and the line starts with no indentation).